packages/docgen/src/engine.js

```diff
@@ -1,25 +1,27 @@
 /**
  * External dependencies.
  */
-const espree = require( 'espree' );
+const babel = require( '@babel/parser' );
 const { flatten } = require( 'lodash' );
 
 /**
  * Internal dependencies.
  */
 const getIntermediateRepresentation = require( './get-intermediate-representation' );
+const babelConfig = require( '../../../babel.config' );
```

Comment:
We shouldn't use files from outside the `packages/docgen` repository, otherwise, it'll fail when used as a standalone npm binary.

Reply:
Then, we can make a command line option like --babel-config. And pass the path to the file in `docs` folder.

How about this?